lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -104,6 +86,7 @@ function filteredAndIndexedByUrl(networkRecords) {
     // Include 404 scripts/links generated by the parser because they are likely blocking.
     if (isHtml || isParserScriptOrStyle || (isFailedRequest && isParserGenerated)) {
       prev[record._url] = {
+        isLinkPreload: record.isLinkPreload,
```

Comment:
this is a little new.

CL: https://codereview.chromium.org/2751023005
cr rev: 460094

it's not yet in stable chrome (58), but its in 59.

how do you want to handle stable?